database/repo/get.go

```diff
@@ -5,21 +5,21 @@ package repo
 import (
 	"context"
 
+	api "github.com/go-vela/server/api/types"
 	"github.com/go-vela/types/constants"
-	"github.com/go-vela/types/database"
-	"github.com/go-vela/types/library"
 )
 
 // GetRepo gets a repo by ID from the database.
-func (e *engine) GetRepo(ctx context.Context, id int64) (*library.Repo, error) {
+func (e *engine) GetRepo(ctx context.Context, id int64) (*api.Repo, error) {
 	e.logger.Tracef("getting repo %d from the database", id)
 
 	// variable to store query results
-	r := new(database.Repo)
+	r := new(Repo)
 
 	// send query to the database and store result in variable
 	err := e.client.
 		Table(constants.TableRepo).
+		Preload("Owner").
```

Comment:
i wonder if this should be a constant somewhere, like the table name references

Reply:
```suggestion
		Preload(constants.PreloadsOwner).
```

Reply:
Going to implement this idea when `go-vela/types/constants` is migrated